lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -148,7 +148,7 @@ DefParameterType('GeneralText', sub {
     my ($gullet) = @_;
     my $open = $gullet->readXToken;
     if ($open->equals(T_BEGIN)) {
-      return $gullet->readBalanced; }
+      return scalar($gullet->readBalanced); }
```

Comment:
I am a bit stumped at this line. Returning the number of tokens instead of the tokens themselves? From within a parameter type ?!?!? Is this some sort of intermediate code that slipped by or am I missing something extremely obvious?

Reply:
Good news is that all tests pass with the scalar removed. I'm tempted to just push in the change since it looks so horrendous.

Reply:
Done.